compose/cli/command.py

```diff
@@ -115,9 +116,21 @@ def unicode_paths(paths):
     return None
 
 
+def get_profiles_from_options(options, environment):
+    profile_option = options.get('--profile')
+    if profile_option:
+        return profile_option
+
+    profiles = environment.get('COMPOSE_PROFILE')
+    if profiles:
+        return profiles.split(',')
```

Comment:
Since the profile names are not paths or file names and there should be no difference in specifying profiles on different OS I feel this to be semantically incorrect.
But since I have no strong opinion on this I can implement this if it is the consensus.

Reply:
From my understanding, a profile is a service label that we use to filter services, not a file path. I wouldn't over-complicate things by enabling a custom separator. I think going with `,` should suffice.

Reply:
OK. I agree with you both :smiley: 
My thoughts were more around the idea of using an homogeneous separator. 
Comma is fine for me.